src/main/java/net/sf/jabref/logic/util/strings/StringUtil.java

```diff
@@ -500,4 +500,35 @@ private static String removeSingleBracesAroundCapitals(String s) {
     public static String unifyLineBreaksToConfiguredLineBreaks(String s) {
         return LINE_BREAKS.matcher(s).replaceAll(Globals.NEWLINE);
     }
+
+    /**
+     * Optimized method for converting a String into an Integer
+     *
+     * From http://stackoverflow.com/questions/1030479/most-efficient-way-of-converting-string-to-integer-in-java
+     *
+     * @param str the String holding an Integer value
+     * @throws NumberFormatException if str cannot be parsed to an int
+     * @return the int value of str
+     */
+    public static int intValueOf(String str) {
```

Comment:
Do we really call this method so often that a optimized version is needed? `Integer.parseInt()` seems to be efficient enough.